### PR TITLE
added a template for community visualizations

### PR DESCRIPTION
**Pull request by y3l2n**
Added a template with README for community visualizations

### CODE REVIEW

File: templates/community-viz/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "config": {
+    "gcsDevBucket": "",
```
Comment (diminishedprime) [inline templates/community-viz/package.json]:
```suggestion
    "gcsDevBucket": "{{DEV_BUCKET}}",
```

File: templates/community-viz/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "gcsProdBucket": "",
```
Comment (diminishedprime) [inline templates/community-viz/package.json]:
```suggestion
    "gcsProdBucket": "{{PROD_BUCKET}}",
```

File: templates/community-viz/src/index.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+line {
+  stroke: #E8EAED;
+  stroke-width: 1px;
+}
+
+circle {
+  opacity: 0.8;
+  stroke-width: 1;
+
```
Comment (diminishedprime) [inline templates/community-viz/src/index.css]: nit here and throughout. Could you get rid of extra newlines.

File: templates/community-viz/src/index.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  // append a div to the DOM
+  if (document.querySelector('div')) {
+    var oldDiv = document.querySelector('div');
+    oldDiv.parentNode.removeChild(oldDiv);
+  }
+
+  // append the data to a div
+  let div = document.createElement('div');
+  var msgText = JSON.stringify(data, null, 2);
+  div.innerHTML = `<pre>${msgText}</pre>`;
```
Comment (diminishedprime) [inline templates/community-viz/src/index.js]:
```suggestion
  div.innerHTML = `<pre>export const message = {${msgText}};</pre>`;
```

File: templates/community-viz/src/index.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+          "options": {
+            "min": 1,
+            "max": 1
+          }
+        }
+      ]
+    }
+  ],
+  "style": [ ]
+}
```
Comment (diminishedprime) [inline templates/community-viz/src/index.json]: nit: remove extra whitespace.

File: templates/community-viz/webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+//
```
Comment (diminishedprime) [inline templates/community-viz/webpack.config.js]: Should there be a comment here?

File: templates/community-viz/README.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
+# Data Studio Community Visualization Local Development Template
+
+Data Studio [community visualizations][community viz] allow you to write custom
```
Comment (diminishedprime) [inline templates/community-viz/README.md]:
```suggestion
Data Studio [community visualizations][community viz] allows you to write custom
```

File: templates/community-viz/README.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+JavaScript visualizations for [Google Data Studio][datastudio].
+
+## About this template
+
+This template provides
+1. An opinionated template for developing community visualizations
+2. A local development and feedback workflow
```
Comment (diminishedprime) [inline templates/community-viz/README.md]:
```suggestion
1. A local development and feedback workflow
```

File: templates/community-viz/README.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+3. Build and deployment scripts
```
Comment (diminishedprime) [inline templates/community-viz/README.md]:
```suggestion
1. Build and deployment scripts
```

File: templates/community-viz/README.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+| Config     | [src/index.json]    | [config reference]     |
+| JavaScript | [src/index.js]      | [write viz code]       |
+| CSS        | [src/index.css]     | [write css code]       |
+
+## Using this template
+
+To create a new community visualization using this template, run the command
+
+```bash
+npx @google/dscc-gen
```
Comment (diminishedprime) [inline templates/community-viz/README.md]:
```suggestion
npx @google/dscc-gen --project_choice community-viz
```

File: templates/community-viz/webpack.config.js (hunk 6 above)
Comment (y3l2n) [reply templates/community-viz/webpack.config.js]: Nope

File: templates/community-viz/src/index.json (hunk 5 above)
Comment (y3l2n) [reply templates/community-viz/src/index.json]: I ran prettier, which should fix these things.

**Pull request merged**